eq-author/src/App/QuestionnairesPage/QuestionnairesView/index.test.js

```diff
@@ -373,5 +373,64 @@ describe("QuestionnairesView", () => {
         expect(getByText("2 of 2")).toBeTruthy();
       });
     });
+    describe("Sorting", () => {
+      let props;
+      beforeEach(() => {
+        const questionnaires = [
+          buildQuestionnaire(4),
+          buildQuestionnaire(2),
+          buildQuestionnaire(1),
+          buildQuestionnaire(3),
+          buildQuestionnaire(5),
+        ];
+        const user = {
+          id: "3",
+          name: "Foo",
+          email: "foo@bar.com",
+          displayName: "Foo",
+        };
+
+        props = {
+          questionnaires,
+          user,
+          onDeleteQuestionnaire: jest.fn(),
+          onDuplicateQuestionnaire: jest.fn(),
+          onCreateQuestionnaire: jest.fn(),
+        };
+      });
+      afterEach(() => {
+        window.localStorage.removeItem(STORAGE_KEY);
+      });
+      it("should sort by title descending when title header is clicked", () => {
```

Comment:
Can we assert the starting order as well just so we know it has done something. Or maybe add a test for the default order?

Reply:
Added a test for the default order.